apps/meteor/app/livechat/server/api/lib/inquiries.ts

```diff
@@ -39,24 +40,25 @@ export async function findInquiries({
 	pagination: { offset: number; count: number; sort: Record<string, number> };
 }): Promise<PaginatedResult<{ inquiries: Array<ILivechatInquiryRecord> }>> {
 	const department = await applyDepartmentRestrictions(userId, filterDepartment);
-
+	const defaultSort = await LivechatInquiry.getSortingQuery();
 	const options = {
 		limit: count,
-		sort: sort || { ts: -1 },
 		skip: offset,
+		sort: { ...sort, ...(defaultSort as object) },
 	};
-
-	const filter = {
-		...(status && { status }),
+	const filter: Filter<ILivechatInquiryRecord> = {
 		$or: [
 			{
 				$and: [{ defaultAgent: { $exists: true } }, { 'defaultAgent.agentId': userId }],
 			},
-			{ ...(department && { department }) },
+			{ ...((!!department && { department }) || {}) },
```

Comment:
Do we need this change? What's fixing? 👀

Reply:
I'm not sure y that was added but it seems to be not changing the existing logic, so I'll revert it